### PR TITLE
Update markdown Trafikverket Train

### PR DESCRIPTION
**Pull request by gjohansson-ST**
## Proposed change

Update markdown for Trafikverket Train to provide better text.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/trafikverket_train.markdown [hunk 1]
```diff
@@ -20,21 +20,23 @@ Retrieve train departure information from [Trafikverket](https://www.trafikverke
...
 - Canceled status.
-- The number of minutes delayed.
+- Delayed time
 - Planned time if no delays occur.
 - Estimated time of arrival if delays occur.
 - Actual time - when it did arrive.
 - Other information / additional texts.
 - Deviations.
 
+Next departure is calculated from actual, estimated and planned to provide most accurate information about departure.
```
Comment (davet2001) [inline source/_integrations/trafikverket_train.markdown]: Add 'the' after 'provide'.

**Review by davet2001 — CHANGES_REQUESTED** [review]

**Review by davet2001 — DISMISSED** [review]
LGTM 👍

File: source/_integrations/trafikverket_train.markdown [hunk 2]
```diff
@@ -20,21 +20,23 @@ Retrieve train departure information from [Trafikverket](https://www.trafikverke
...
+- Retrieve the next departure between two stations.
+- Retrieve information for specific departure time between two stations.
 - Set up an alert or perform actions if your train is delayed or canceled.
 
 ## Retrieved data
 
 - Next departure for the specific train line.
 - Canceled status.
-- The number of minutes delayed.
+- Delayed time
```
Comment (frenck) [inline source/_integrations/trafikverket_train.markdown]:
```suggestion
- Delayed time.
```

File: source/_integrations/trafikverket_train.markdown [hunk 3]
```diff
@@ -20,21 +20,23 @@ Retrieve train departure information from [Trafikverket](https://www.trafikverke
...
 - Planned time if no delays occur.
 - Estimated time of arrival if delays occur.
 - Actual time - when it did arrive.
 - Other information / additional texts.
 - Deviations.
 
+Next departure is calculated from actual, estimated and planned to provide the most accurate information about departure.
```
Comment (frenck) [inline source/_integrations/trafikverket_train.markdown]:
```suggestion
The next departure is calculated from actual, estimated, and planned to provide the most accurate information about departure.
```

**Review by frenck — APPROVED** [review]
Thanks,  @gjohansson-ST 👍

**Pull request merged**